python/brayns/cli/core/camera_cli.py

```diff
@@ -44,23 +44,23 @@ def register(self, parser: argparse.ArgumentParser) -> None:
             '--camera_type',
             choices=[camera.name for camera in self.available],
             default=self.name,
-            help='Camera type',
+            help='Type of camera to use for render',
         )
         parser.add_argument(
```

Comment:
Why not options to set the camera as position + target as well? Its the most common approach in computer graphics.